http_call.go

```diff
@@ -267,7 +364,7 @@ func (d *duplexHTTPCall) makeRequest() {
 	}
 	// Once we send a message to the server, they send a message back and
 	// establish the receive side of the stream.
-	response, err := d.httpClient.Do(d.request) //nolint:bodyclose
+	response, err := d.httpClient.Do(d.request)
```

Comment:
I think it's definitely preferable to have this close to the code in question instead of moving them far away into the golang config. I think the golang config is a better place for "cross-cutting" rules -- like disable "forcetypeassert" checks in `*_test.go` files.